### DISCUSSION TITLE
IDEA Debugger 模式下“翻译文档”

### DISCUSSION THREAD

**Discussion by ghost** [YiiGuxing/TranslationPlugin#1680] [Q&A] [answered]
在 Debug 模式下鼠标聚焦到引用上无法自动翻译，只能自己手动选中 javadoc 翻译文档希望可以改进一下

**Comment by YiiGuxing** [comment, marked as answer]
@ghost
在 Debug 模式下鼠标聚焦到引用上IDE本身就不弹出文档窗口啊，不弹出就没办法翻译嘛，在这个模式下可以使用查看快速文档的快捷键弹出文档窗口，这个时候是会自动翻译的。
